### PR TITLE
Mobile responsive

### PR DESCRIPTION
**Pull request by jeetbhalani15**
##  Added Mobile-Responsive

### CODE REVIEW

**Review by Jay-Gadhiya — APPROVED** [review]

File: Gravity-Notes/src/Components/Navigation/Navbar.js [hunk 1]
```diff
@@ -38,11 +47,17 @@ export const Navbar = ()=>{
             <div className="widgets">
                 <div className="refresh my-3"><button className='nav-btn'> <a href="#"> <MdRefresh size={30}/></a></button></div>
                 <div className="view my-3 "><button className='nav-btn'><a href="#"><MdGridView size={30}/></a></button></div>
-                <div className="settings my-3 "><button className='nav-btn'><a href="#"><IoSettingsOutline size={30}/></a></button></div>
+                {/* <div className="settings my-3 "><button className='nav-btn'><a href="#"><IoSettingsOutline size={30}/></a></button></div> */}
```
Comment (ckahub) [inline Gravity-Notes/src/Components/Navigation/Navbar.js:50]: delete coment code

File: Gravity-Notes/src/Components/TextArea.js/Textarea.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .textarea-form{
     position: relative;
-    width: 600px;
```
Comment (ckahub) [inline Gravity-Notes/src/Components/TextArea.js/Textarea.css:3]: use rem everywhere

**Review by ckahub — APPROVED** [review]
change title to gravity notes

**Review by ark2002 — COMMENTED** [review]
that looks good

File: Gravity-Notes/src/Components/TextArea.js/Textarea.css (hunk 2 above)
Comment (prath-dev) [reply Gravity-Notes/src/Components/TextArea.js/Textarea.css:3]: Do not use px as vikram sir also said that above 10px we should always use rem .

Comment (jeetbhalani15) [reply Gravity-Notes/src/Components/TextArea.js/Textarea.css:3]: made changes

**Pull request merged**